src/pages/admin/categories/CategoryList/index.module.scss

```diff
@@ -1,144 +1,133 @@
-$light-green: #E0EAEC;
-$light-grayish-blue: #ced4da;
-$gray: rgba(255, 255, 255, 0);
-$light-bluegreen: #9abec1;
-$very-dark-grayish-blue: #48535b;
-$style: none;
-$sizefirst: 24px;
-$sizesec: 18px;
-$sizethrd: 14px;
-$white: #ffffff;
-$cnt: center;
-$flx: flex;
+@import '../../../../styles/globalVariables';
```

Comment:
please import it at the root index.js so that we will not be importing it in every file